src/QMCWaveFunctions/Jastrow/JastrowBuilder.cpp

```diff
@@ -163,6 +167,12 @@ bool JastrowBuilder::addTwoBody(xmlNodePtr cur)
 {
   ReportEngine PRE(ClassName,"addTwoBody(xmlNodePtr)");
   bool success=false;
+  RadialJastrowBuilder rb(targetPtcl,targetPsi);
+  return rb.put(cur);
+}
+
+
+  /*
```

Comment:
Are these commented part all moved to the new builder? If not, please add a comment.

Reply:
All of this was moved to the new builder.  I have removed the comments.